proxysql-admin

```diff
@@ -3152,6 +3199,9 @@ function main() {
   elif [[ $REPORT_STATUS -eq 1 ]]; then
 
     report_status $WRITER_HOSTGROUP_ID
+  elif [[ $UPDATE_MYSQL_VERSION -eq 1 ]]; then
```

Comment:
formatting, should be an empty line before the elif

Reply:
Done